src/main/kotlin/app/revanced/patches/music/misc/microg/shared/Constants.kt

```diff
@@ -0,0 +1,7 @@
+package app.revanced.patches.music.misc.microg.shared
+
+object Constants {
+    internal const val BASE_MICROG_PACKAGE_NAME = "com.mgoogle"
+    internal const val REVANCED_MUSIC_APP_NAME = "YT Music ReVanced"
```

Comment:
The brand name should not be morphed:
```suggestion
    internal const val REVANCED_MUSIC_APP_NAME = "YouTube Music ReVanced"
```